server/src/test/java/io/crate/planner/SelectPlannerTest.java

```diff
@@ -1509,4 +1509,17 @@ public void test_non_recursive_nested_with() throws Exception {
 
         assertThat(withPlan).isEqualTo(subSelectPlan);
     }
+
+    @Test
+    public void test_filter_is_pushed_down_to_collect_with_view() throws Exception {
```

Comment:
maybe:
```suggestion
    public void test_filter_on_object_col_is_pushed_down_to_collect_with_view() throws 
    Exception {
```
to be more clear?